### PR TITLE
dumpe2fs: add page

### PR DESCRIPTION
**Pull request by teeteejo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/dumpe2fs.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# dumpe2fs
+
+> Print the super block and blocks group information for ext2/ext3/ext4 filesystem.
+> Unmount the partition before running this command using `umount {{device}}`.
+> More information: <https://linux.die.net/man/8/dumpe2fs>.
+
+- Display ext2, ext3 and ext4 filesystem information for the device `/dev/sda1`:
+
+`dumpe2fs {{/dev/sda1}}`
```
Comment (navarroaxel) [inline pages/linux/dumpe2fs.md:9]:
```suggestion
`dumpe2fs {{/dev/sdXN}}`
```

File: pages/linux/dumpe2fs.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Display the blocks which are reserved as bad in the filesystem:
+
+`dumpe2fs -b {{/dev/sda1}}`
```
Comment (navarroaxel) [inline pages/linux/dumpe2fs.md:13]:
```suggestion
`dumpe2fs -b {{/dev/sdXN}}`
```

File: pages/linux/dumpe2fs.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- Force display filesystem information even with non-recogonisable feature flags:
+
+`dumpe2fs -f {{/dev/sda1}}`
```
Comment (navarroaxel) [inline pages/linux/dumpe2fs.md:17]:
```suggestion
`dumpe2fs -f {{/dev/sdXN}}`
```

File: pages/linux/dumpe2fs.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Only display the superblock information and not any of the block group descriptor detail information:
+
+`dumpe2fs -h {{/dev/sda1}}`
```
Comment (navarroaxel) [inline pages/linux/dumpe2fs.md:21]:
```suggestion
`dumpe2fs -h {{/dev/sdXN}}`
```

File: pages/linux/dumpe2fs.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+- Print the detailed group information block numbers in hexadecimal format:
+
+`dumpe2fs -x {{/dev/sda1}}`
```
Comment (navarroaxel) [inline pages/linux/dumpe2fs.md:25]:
```suggestion
`dumpe2fs -x {{/dev/sdXN}}`
```

File: pages/linux/dumpe2fs.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# dumpe2fs
+
+> Print the super block and blocks group information for ext2/ext3/ext4 filesystem.
+> Unmount the partition before running this command using `umount {{device}}`.
+> More information: <https://linux.die.net/man/8/dumpe2fs>.
+
+- Display ext2, ext3 and ext4 filesystem information for the device `/dev/sda1`:
```
Comment (navarroaxel) [inline pages/linux/dumpe2fs.md:7]:
```suggestion
- Display ext2, ext3 and ext4 filesystem information:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by teeteejo** [comment]
Thank you @navarroaxel

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
@sbrl, @owenvoke: could you please review?

**Review by sbrl — APPROVED** [review]
Thanks, @teeteejo!

Just made a really minor grammar fixx.

**Pull request merged**